Library/Homebrew/dev-cmd/determine-test-runners.rb

```diff
@@ -0,0 +1,259 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "cli/parser"
+require "formula"
+
+class TestRunnerFormula
```

Comment:
I think putting this in a `test_runner_formula.rb` would be nice.

Reply:
Yup, I'll do that when refactoring stuff here into a separate module. Just fixing the generic OS test failure first.